InclusionBot.md

```diff
@@ -0,0 +1,30 @@
+# Inclusion Bot change history
```

Comment:
I wrestled with this. Maybe it doesn't matter who the target audience is, but I had envisioned this as being a history of the bot as a product rather than its technical history, if that makes sense? I don't feel strongly about how the file is named, so long as the history is captured somewhere. We can always link to it in the README so folks don't have to be aware of naming conventions.

So does `CHANGELOG.inclusion_bot.md` and an update to the README sound good?